site/zenodo_rdm/files.py

```diff
@@ -149,3 +151,21 @@ def send_file(
 def storage_factory(**kwargs):
     """Create custom storage factory to enable file offloading."""
     return pyfs_storage_factory(filestorage_class=EOSFilesOffload, **kwargs)
+
+
+def checksum_verification_files_query():
+    """Return a FileInstance query taking into account file URI prefixes."""
+    files = FileInstance.query
+
+    uri_prefixes = current_app.config.get(
+        "FILES_REST_CHECKSUM_VERIFICATION_URI_PREFIXES"
+    )
+    if uri_prefixes:
+        files = files.filter(
+            sa.or_(*[FileInstance.uri.startswith(p) for p in uri_prefixes])
+        )
+    # Filtering out .ptif media files
+    included_path = current_app.config["FILES_STORAGE_BASE_PATH"]
+    files = files.filter(FileInstance.uri.startswith(included_path))
```

Comment:
I had in mind using the `FILES_REST_CHECKSUM_VERIFICATION_URI_PREFIXES` config entirely, so no need to have another config variable.